src/vs/workbench/services/textfile/common/textFileEditorModel.ts

```diff
@@ -334,6 +356,9 @@ export class TextFileEditorModel extends BaseTextEditorModel implements ITextFil
 				diag('makeDirty() - prevented save because we are in conflict resolution mode', this.resource, new Date());
 			}
 		}
+
+		// Trigger listeners only if no auto save occurred and model changes were not blocked
+		this._onDidContentChange.fire();
```

Comment:
The fact that it doesn't trigger on a revert is utilized by `BackupModelService`, should I maybe make `onDidContentChange` an `Event<StateChange>` instead so they can be differentiated? Or would changing the name to something else be better?

The comment was wrong, I guess I was thinking of https://github.com/Microsoft/vscode/issues/15129 but this is going to be handled within one of the backup services. Removed in https://github.com/Microsoft/vscode/pull/14115/commits/d346ed2d7252cf1d3654abd86b256f89c4d9d6b3